libra-swarm/src/main.rs

```diff
@@ -98,6 +98,15 @@ fn main() {
         validator_set_file,
         faucet_key_file_path,
     );
+    println!("To run transaction generator run:");
+    println!(
+        // WIP(ankush): Do we need workplace for local cluster-test?
+        "\tcargo run --bin cluster-test -- --mint-file {:?} --peers localhost:{}  --emit-tx --workplace <workplace-name>",
```

Comment:
Question: Do we need a `--workplace` parameter when we run cluster-test against a libra-swarm?

Reply:
Nope, `--workplace` only makes sense in AWS setup (it is used for discovery)

Reply:
But perhaps we do want to put all the peers here, not just one in `--peers`